driver/modern_bpf/helpers/interfaces/syscalls_dispatcher.h

```diff
@@ -13,14 +13,42 @@
 #include <helpers/base/read_from_task.h>
 #include <helpers/extract/extract_from_kernel.h>
 
+// We don't want to send DROP_E/DROP_X events from the enter tracepoint because it would requires us
+// to create a dedicated tail table for the enter. It is enough to send DROP_E/DROP_X events from
+// the exit tracepoint.
+static __always_inline bool syscalls_dispatcher__sampling_logic_enter(uint32_t syscall_id) {
```

Comment:
Moved from `driver/modern_bpf/programs/attached/dispatchers/syscall_enter.bpf.c` as it is shared by `sys_enter` dispatcher and TOCTOU mitigation progs.